mediacapture-streams/enumerating-local-media-devices/api-present.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+<title>enumerateDevices: test that enumerateDevices is present</title>
+<link rel="author" title="Dr Alex Gouaillard" href="mailto:agouaillard@gmail.com"/>
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#enumerating-devices">
+<meta name='assert' content='Check that the enumerateDevices() method is present.'/>
+<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
```

Comment:
shall i modify them all then?

grep -R link * | grep testharness
enumerating-local-media-devices/api-present.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
mediastreams-as-media-elements/video-assignment-manual.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/mediadevices/api-present.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/navigatorusermedia/api-present.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/navigatorusermedia/deny.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/navigatorusermedia/empty-option-param.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/navigatorusermedia/getusermedia-impossible-constraint.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/navigatorusermedia/getusermedia-optional-constraint.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/navigatorusermedia/getusermedia-trivial-constraint.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
obtaining-local-multimedia-content/navigatorusermedia/unknownkey-option-param.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/introduction/disabled-audio-silence.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/introduction/disabled-video-black.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/audio.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/mediastream-addtrack.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/mediastream-finished-add.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/mediastream-gettrackid.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/mediastream-id-manual.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/mediastream-idl.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/mediastream-removetrack.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/stream-ended.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastream/video.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastreamtrack/mediastreamtrack-end.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastreamtrack/mediastreamtrack-id.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/mediastreamtrack/mediastreamtrack-init.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/video-and-audio-tracks/audiostreamtrack.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
stream-api/video-and-audio-tracks/videostreamtrack.html:<link rel='stylesheet' href='/resources/testharness.css' media='all'/>